include/openmc/volume_calc.h

```diff
@@ -8,21 +8,31 @@
 
 #include <string>
 #include <vector>
+#include <gsl/gsl>
 
 namespace openmc {
 
+enum class VolumeTriggerMetric {
+  NONE = 0,
+  VARIANCE = 1,
+  STD_DEV = 2,
+  REL_ERR = 3
```

Comment:
I would just make this consistent with what appears in `include/openmc/tallies/trigger.h`.

Reply:
Actually, is there any reason we can't just use the existing `TriggerMetric` enum?

Reply:
Nope! Updated .